spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -729,8 +729,7 @@ describe('document-capture/components/acuant-capture', () => {
     const input = getByLabelText('Image');
     userEvent.upload(input, validUpload);
 
-    await new Promise((resolve) => addPageAction.callsFake(resolve));
-    expect(addPageAction).to.have.been.calledWith({
+    await expect(addPageAction).to.eventually.be.calledWith({
```

Comment:
we could just return the promise instead of awaiting it right? and wouldn't need to be an `async` function anymore?

Reply:
> we could just return the promise instead of awaiting it right? and wouldn't need to be an `async` function anymore?

Yeah, it'd essentially be the same. I'd opted for this because (a) it makes it easier to revise a function later to add after an `await` than it is to refactor a promise-returning function and (b) it was already async and this was fewer code changes.